packages/typespec-client-generator-core/test/decorators.test.ts

```diff
@@ -3027,4 +3027,311 @@ describe("typespec-client-generator-core: decorators", () => {
       strictEqual(uploadOp.parameters[0].correspondingMethodParams[0], blobName);
     });
   });
+
+  describe("scope negation", () => {
+    it("single scope negation", async () => {
+      const runnerWithCSharp = await createSdkTestRunner({
+        emitterName: "@azure-tools/typespec-csharp",
+      });
+      await runnerWithCSharp.compile(`
+        @service
+        @test namespace MyService {
```

Comment:
`@test` is no need if you don't want to get the raw type. also could you remove the `@access`

Reply:
refined